### ISSUE TITLE
[BUG]: Colossalai-OpenMoE-8b : loss value is very large and cannot converge

### ISSUE THREAD

**Issue by hangchen426926** [hpcaitech/ColossalAI#5212]
### 🐛 Describe the bug

I am currently running the Colossalai/examples/language/openmoe project with the following experimental setup:

datasets: load_dataset("yizhongw/self_instruct/data/finetuning/self_instruct_221203", "super_natural_instructions"), 
model: openmoe-8b
A100_num_GPU：8
Epoch：3
BatchSize：4
LR：0.00001
zero_stage：1
precision：bf16
Boost plugin：ep_zero
extra_dp_size：2
max_length：2048

**### Issue : loss value is very large and cannot converge**
I've encountered challenges during the training process where the convergence seems unachievable. The training loss value persists at an exceptionally high level (exceeding 2.3E+10) even after running 3 epochs. The logged information provided below showcases loss values:
<img width="539" alt="loss" src="https://github.com/hpcaitech/ColossalAI/assets/56995618/fcea6b2c-eea9-4071-9294-c9cc86ccfa2c">

Furthermore, in the default setup of the openmoe project, the lr is 0.00001. Considering this, I suspect the issue might stem from an overfitting problem. Consequently, I attempted to adjustment the lr parameter(from 0.00001 to 0.0000001) ,but still encountered the same problem.
![tempsnip](https://github.com/hpcaitech/ColossalAI/assets/56995618/5d0000b4-228d-431f-a66c-40b19d21baf7)
![tempsnip2](https://github.com/hpcaitech/ColossalAI/assets/56995618/402dfa58-a3d7-4ead-9c17-342324f28127)

### Environment

torch 1.13.1
python 3.8.17
cuda:11.7.17

**Comment by Orion-Zheng** [comment]
Thank you for your valuable feedback! 😃We are working on this bug and will get back to you later.

**Comment by noob-ctrl** [comment]
I also encountered this problem

**Comment by noob-ctrl** [comment]
@Orion-Zheng Has this bug been solved?

**Comment by obananas** [comment]
@Orion-Zheng  Has this bug been solved?
